codespell_lib/data/dictionary.txt

```diff
@@ -12340,6 +12346,7 @@ famlily->family
 famoust->famous
 fanatism->fanaticism
 fancyness->fanciness
+farction->faction, fraction,
```

Comment:
I'd probably favour this:
```suggestion
farction->fraction, faction,
```

Grep.app seems to agree.

Can you also add this (but in rare as it's also a medical term):
`infarction->infraction`